applications/PfemFluidDynamicsApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -183,6 +184,12 @@ void AddCustomProcessesToPython(pybind11::module &m)
         .def(py::init<ModelPart &, Parameters &>())
         .def(py::init<ModelPart &, const Variable<array_1d<double, 3>> &, array_1d<double, 3> &>())
         .def("Execute", &AssignVectorVariableToPfemConditionsProcess::Execute);
+
+	py::class_<UpdateConditionsOnFreeSurfaceProcess, UpdateConditionsOnFreeSurfaceProcess::Pointer, ProcessBaseType>(m, "UpdateConditionsOnFreeSurfaceProcess")
+	    .def(py::init<ModelPart &, Parameters>())
+        .def(py::init<ModelPart &, Parameters &>())
```

Comment:
```suggestion

```

due to the internal memory management `Parameters` should be passed by value (always)